### PR TITLE
doc: Update docs with nRF9151 where the support is added

### PR DESCRIPTION
**Pull request by divipillai**
Support for nRF9151 is added to following:
Hence, docs are updated accordingly:

* Asset tracker applicaiton
* Memfault sample
* All networking samples

Also, updated the connectivity bridge readme.txt to mention
Bluetooth® LE instead of BLE and added the trademark.

### CODE REVIEW

**Comment by divipillai** [comment]
dependent on PR https://github.com/nrfconnect/sdk-nrf/pull/13810.

**Review by maxd-nordic — APPROVED** [review]
lgtm

**Review by greg-fer — COMMENTED** [review]
Please also update ap_protect.rst

**Comment by divipillai** [comment]
> Please also update ap_protect.rst

This PR scope is just for the asset tracker application, Memfault sample, and the networking samples. I will be creating a PR for other NCS pages in general that need an update for nRF9151.

File: applications/connectivity_bridge/boards/thingy91_nrf52840_readme.txt [hunk 1]
```diff
@@ -15,13 +15,13 @@ COM Ports
 This USB interface exposes two COM ports mapped to the physical UART interfaces between the nRF91 Series and nRF52840 devices.
 When opening these ports manually (without using the LTE Link Monitor), be aware that the USB COM port baud rate selection is applied to the UART.
 
-Bluetooth LE Central UART Service
-=================================
+Bluetooth® LE Central UART Service
```
Comment (coderbyheart) [inline applications/connectivity_bridge/boards/thingy91_nrf52840_readme.txt:18]: **problem:** unrelated changed, not mentioned in commit message.

File: applications/connectivity_bridge/boards/thingy91_nrf52840_readme.txt [hunk 2]
```diff
@@ -15,13 +15,13 @@ COM Ports
...
+==================================
 
 This device advertises as "Thingy:91 UART".
 Connect using a Bluetooth LE Central device, for example a phone running the nRF Connect app:
 https://www.nordicsemi.com/Software-and-tools/Development-Tools/nRF-Connect-for-mobile/
 
-NOTE: The BLE interface is unencrypted and intended to be used during debugging.
+NOTE: The Bluetooth LE interface is unencrypted and intended to be used during debugging.
```
Comment (coderbyheart) [inline applications/connectivity_bridge/boards/thingy91_nrf52840_readme.txt:25]: **problem:** unrelated changed, not mentioned in commit message.

File: applications/connectivity_bridge/boards/thingy91x_nrf5340_cpuapp_readme.txt [hunk 3]
```diff
@@ -14,13 +14,13 @@ When opening these ports manually (without using LTE Link Monitor) be aware that
 This USB interface exposes two COM ports mapped to the physical UART interfaces between the nRF91 Series and nRF5340 devices.
 When opening these ports manually (without using the LTE Link Monitor), be aware that the USB COM port baud rate selection is applied to the UART.
 
-Bluetooth LE Central UART Service
-=================================
+Bluetooth® LE Central UART Service
```
Comment (coderbyheart) [inline applications/connectivity_bridge/boards/thingy91x_nrf5340_cpuapp_readme.txt:17]: **problem:** unrelated changed, not mentioned in commit message.

File: applications/connectivity_bridge/boards/thingy91x_nrf5340_cpuapp_readme.txt [hunk 4]
```diff
@@ -14,13 +14,13 @@ When opening these ports manually (without using LTE Link Monitor) be aware that
...
+==================================
 
 This device advertises as "Thingy:91 X UART".
 Connect using a Bluetooth LE Central device, for example a phone running the nRF Connect app:
 https://www.nordicsemi.com/Software-and-tools/Development-Tools/nRF-Connect-for-mobile/
 
-NOTE: The BLE interface is unencrypted and intended to be used during debugging.
+NOTE: The Bluetooth LE interface is unencrypted and intended to be used during debugging.
```
Comment (coderbyheart) [inline applications/connectivity_bridge/boards/thingy91x_nrf5340_cpuapp_readme.txt:24]: **problem:** unrelated changed, not mentioned in commit message.

**Review by coderbyheart — CHANGES_REQUESTED** [review]

File: applications/connectivity_bridge/boards/thingy91_nrf52840_readme.txt (hunk 1 above)
Comment (divipillai) [reply applications/connectivity_bridge/boards/thingy91_nrf52840_readme.txt:18]: These changes were added later. Updated the commit messages. Thanks Markus.

File: samples/net/http_server/README.rst [hunk 5]
```diff
@@ -333,6 +334,8 @@ Troubleshooting
 If you have issues with connectivity on nRF91 Series devices, see the `Cellular Monitor`_ documentation to learn how to capture modem traces to debug network traffic in Wireshark.
 Modem traces can be enabled by providing a snippet with the west build command using the ``-S`` argument:
 
+The following is an example for the nRF9161 DK:
```
Comment (umapraseeda) [inline samples/net/http_server/README.rst:337]:
```suggestion
Modem traces can be enabled by providing a snippet with the west build command using the ``-S`` argument as shown in the following example for nRF9161 DK:

```

**Review by coderbyheart — APPROVED** [review]

**Review by umapraseeda — APPROVED** [review]

**Pull request merged**